sudoku/easy/script.js

```diff
@@ -0,0 +1,55 @@
+import { addRandom } from '../general/addRandom.js';
+import { checkGrid, checkRowsArr, checkGridArr, checkColumnsArr } from './checkFunctionsRandom.js';
+import { start, pause, reset } from '../general/stopwatch.js';
+
+export let answers = [];
+export let check = document.querySelector('.check');
+export let startGamebtn = document.querySelector('.change');
+let div = document.querySelectorAll('.board>div');
+let matrix = [];
+let arr = [];
```

Comment:
It looks like you are not changing this array in the future. You can convert them to const
```suggestion
const matrix = [];
const arr = [];
```